layouts/partials/custom-header.html

```diff
@@ -1 +1,2 @@
-<link href="/css/style.css" rel="stylesheet">
+{{ $style := resources.Get "css/style.css" | resources.Minify | resources.Fingerprint }}
+<link rel="stylesheet" href="{{ $style.Permalink }}" />
```

Comment:
Using `Permalink` here interacts badly with netlify PR deploys as it's always a fully absolute path.

I think we want `RelPermalink` instead?